File: apps/mybookkeeper/backend/app/schemas/inquiries/inquiry_list_response.py
"""Paginated envelope for GET /inquiries — same shape as ListingListResponse."""
from __future__ import annotations

from platform_shared.schemas.pagination import ListResponse

from app.schemas.inquiries.inquiry_summary import InquirySummary


class InquiryListResponse(ListResponse[InquirySummary]):
    pass

File: apps/mybookkeeper/frontend/src/shared/types/inquiry/inquiry-list-response.ts
import type { InquirySummary } from "./inquiry-summary";

/**
 * Paginated envelope returned by GET /inquiries — same shape as
 * ListingListResponse. ``has_more`` lets the frontend hide the "Load more"
 * button on the last page.
 */
export interface InquiryListResponse {
  items: InquirySummary[];
  total: number;
  has_more: boolean;
}
